lntest/itest/lnd_channel_graph_test.go

```diff
@@ -789,6 +850,48 @@ func testUpdateNodeAnnouncement(net *lntest.NetworkHarness, t *harnessTest) {
 	ctxt, cancel := context.WithTimeout(ctxb, defaultTimeout)
 	defer cancel()
 
-	_, err := dave.UpdateNodeAnnouncement(ctxt, invalidNodeAnnReq)
+	_, err = dave.UpdateNodeAnnouncement(ctxt, invalidNodeAnnReq)
 	require.Error(t.t, err, "requests without modifiers should field")
+
+	// Alias too long.
+	invalidNodeAnnReq = &peersrpc.NodeAnnouncementUpdateRequest{
+		Alias: strings.Repeat("a", 50),
+	}
+	ctxt, cancel = context.WithTimeout(ctxb, defaultTimeout)
+	defer cancel()
+
+	_, err = dave.UpdateNodeAnnouncement(ctxt, invalidNodeAnnReq)
+	require.Error(t.t, err, "failed to validate an invalid alias for an "+
+		"update node announcement request")
+
+	// Update Node.
+	newAlias := "new-alias"
+
+	nodeAnnReq := &peersrpc.NodeAnnouncementUpdateRequest{
+		Alias: newAlias,
+	}
+
+	ctxt, _ = context.WithTimeout(ctxb, defaultTimeout)
```

Comment:
These calls should all complete pretty quickly, right? We could create one timeout context at the beginning and re-use it throughout our test (which also makes it a bit more readable).